### PR TITLE
fix(plasma-core/ui/web): Toast centering content

### PR DESCRIPTION
**Pull request by Yakutoc**
Необходимо исправить неправильные значения для **padding** и **border-radius**, так же отцентрировать содержимое компонента Toast.   

:baby_chick: Download canary assets:

[colors--canary.253.3589621865.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.253.3589621865.xml)  
[color_sbermarket_ios-swift--canary.253.3589621865.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.253.3589621865.swift)  
[color_sbermarket_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.253.3589621865.kt)  
[color_sbermarket_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.253.3589621865.ts)  
[color_sberprime_ios-swift--canary.253.3589621865.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.253.3589621865.swift)  
[color_sberprime_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.253.3589621865.kt)  
[color_sberprime_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.253.3589621865.ts)  
[PlasmaTokensColor--canary.253.3589621865.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.253.3589621865.swift)  
[typo_mage_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.253.3589621865.kt)  
[typo_mage_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.253.3589621865.ts)  
[typo_plasma_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.253.3589621865.kt)  
[typo_plasma_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.253.3589621865.ts)  
[typo_ruler_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.253.3589621865.kt)  
[typo_ruler_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.253.3589621865.ts)  
[typo_sage_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.253.3589621865.kt)  
[typo_sage_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.253.3589621865.ts)  
[typo_sbermarket_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.253.3589621865.kt)  
[typo_sbermarket_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.253.3589621865.ts)  
[typo_soulmate_kotlin--canary.253.3589621865.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.253.3589621865.kt)  
[typo_soulmate_react-native--canary.253.3589621865.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.253.3589621865.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.116.2-canary.253.3589621865.0
  npm install @salutejs/plasma-core@1.83.2-canary.253.3589621865.0
  npm install @salutejs/plasma-icons@1.110.2-canary.253.3589621865.0
  npm install @salutejs/plasma-temple@1.114.5-canary.253.3589621865.0
  npm install @salutejs/plasma-ui@1.147.4-canary.253.3589621865.0
  npm install @salutejs/plasma-web@1.144.2-canary.253.3589621865.0
  npm install @salutejs/plasma-sb-utils@0.81.2-canary.253.3589621865.0
  # or 
  yarn add @salutejs/plasma-b2c@1.116.2-canary.253.3589621865.0
  yarn add @salutejs/plasma-core@1.83.2-canary.253.3589621865.0
  yarn add @salutejs/plasma-icons@1.110.2-canary.253.3589621865.0
  yarn add @salutejs/plasma-temple@1.114.5-canary.253.3589621865.0
  yarn add @salutejs/plasma-ui@1.147.4-canary.253.3589621865.0
  yarn add @salutejs/plasma-web@1.144.2-canary.253.3589621865.0
  yarn add @salutejs/plasma-sb-utils@0.81.2-canary.253.3589621865.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-253/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-253/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-253/
  ui storybook: http://plasma.sberdevices.ru/pr-253/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-253/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-253/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-253/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-253/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/temple-storybook/

  </details>

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-253/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-253/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-253/
  ui storybook: http://plasma.sberdevices.ru/pr-253/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-253/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-253/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-253/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-253/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/temple-storybook/

  </details>

File: packages/plasma-core/src/components/Toast/Toast.tsx [hunk 1]
```diff
@@ -15,15 +15,18 @@ export type ToastProps = {
 
 export const StyledRoot = styled(Footnote1)`
     display: inline-flex;
-    padding: 0.75rem 1.25rem;
-    border-radius: 1.25rem;
+    align-items: center;
+    padding: 0.688rem 1.25rem;
```
Comment (Yeti-or) [inline packages/plasma-core/src/components/Toast/Toast.tsx:19]: тут точно такой padding сверху? какое-то значение подозрительное

**Review by Yeti-or — COMMENTED** [review]
Можешь ещё скриншот с иконкой в contentLeft добавить ?

File: packages/plasma-core/src/components/Toast/Toast.tsx (hunk 1 above)
Comment (Yeti-or) [reply packages/plasma-core/src/components/Toast/Toast.tsx:19]: у тебя база 16px

11 / 16 => 0.6875

Comment (Yakutoc) [reply packages/plasma-core/src/components/Toast/Toast.tsx:19]: У нас правило линта не даст поставить такое значение.  

`Expected "0.6875" to be "0.688"   number-max-precision`

**Comment by Yakutoc** [comment]
> Можешь ещё скриншот с иконкой в contentLeft добавить ?

Да конечно добавлю, что то не подумал про это. Спасибо )

File: packages/plasma-core/src/components/Toast/Toast.tsx (hunk 1 above)
Comment (Yeti-or) [reply packages/plasma-core/src/components/Toast/Toast.tsx:19]: его вроде можно настроить на это? https://stylelint.io/user-guide/rules/number-max-precision/#options ?

Ну или завести задачку на настроить, а сейчас сделать как тут?

```
packages/plasma-ui/src/components/Tabs/TabsSlider.tsx
14:    /* stylelint-disable-next-line number-max-precision */

packages/plasma-ui/src/components/Cell/CellListItem.tsx
16:        /* stylelint-disable-next-line number-max-precision */

packages/plasma-ui/src/components/Pickers/PickerDots.tsx
34:        /* stylelint-disable number-max-precision */
42:        /* stylelint-enable number-max-precision */
45:        /* stylelint-disable number-max-precision */
61:        /* stylelint-enable number-max-precision */

packages/plasma-ui/src/helpers/Showcase.tsx
28:    /* stylelint-disable-next-line number-max-precision */
```

Comment (Yakutoc) [reply packages/plasma-core/src/components/Toast/Toast.tsx:19]: @Yeti-or Уточню, нам действительно **нужна** точность до 4-5 знака после запятой? 

Браузер все равно не отрисует прям точно **0.6875** такое значение.

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-253/
  <details>
    <summary>New documentation</summary>
    <br />

    http://plasma.fine.sberdevices.ru/pr/demo-canvas-app-pr-253/

  </details>

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-253/
  ui storybook: http://plasma.sberdevices.ru/pr-253/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-253/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-253/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-253/temple-storybook/
  <details>
    <summary>New documentation</summary>
    <br />

    website: http://plasma.fine.sberdevices.ru/pr/pr-253/
    ui storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/ui-storybook/
    web storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/web-storybook/
    b2c storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/b2c-storybook/
    temple storybook: http://plasma.fine.sberdevices.ru/pr/pr-253/temple-storybook/

  </details>

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-core/src/components/Toast/Toast.tsx (hunk 1 above)
Comment (Yeti-or) [reply packages/plasma-core/src/components/Toast/Toast.tsx]: ну теперь оно четко 80 по высоте

**Review by Yeti-or — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

File: packages/plasma-core/src/components/Toast/ToastBase.component-test.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ describe('plasma-core: Toast', () => {
         cy.matchImageSnapshot();
     });
 
-    it.skip('__content-left', () => {
```
[the remark this answers is not in the document]
Comment (Yakutoc) [reply packages/plasma-core/src/components/Toast/ToastBase.component-test.tsx:21]: @Yeti-or Я тоже задался этим вопросом, но в git history каких-либо комментариев на этот счет нет.

**Pull request merged**